packages/models-library/src/models_library/api_schemas_catalog/services_ports.py

```diff
@@ -26,24 +26,37 @@ class ServicePortGet(BaseModel):
         None,
         description="jsonschema for the port's value. SEE https://json-schema.org/understanding-json-schema/",
     )
+
     model_config = ConfigDict(
         json_schema_extra={
-            "example": {
-                "key": "input_1",
-                "kind": "input",
-                "content_schema": {
-                    "title": "Sleep interval",
-                    "type": "integer",
-                    "x_unit": "second",
-                    "minimum": 0,
-                    "maximum": 5,
+            "examples": [
+                {
+                    "key": "input_1",
+                    "kind": "input",
+                    "content_schema": {
+                        "title": "Sleep interval",
+                        "type": "integer",
+                        "x_unit": "second",
+                        "minimum": 0,
+                        "maximum": 5,
+                    },
+                },
+                {
+                    "key": "output_1",
+                    "kind": "output",
+                    "content_media_type": "text/plain",
+                    "content_schema": {
+                        "type": "string",
+                        "title": "File containing one random integer",
+                        "description": "Integer is generated in range [1-9]",
+                    },
                 },
-            }
+            ]
         }
     )
 
     @classmethod
-    def from_service_io(
+    def from_domain_model(
```

Comment:
domain = internal ?
somehow this name feels vague... especially since the other models are in `services_io` module...

Reply:
this is how we call to the adapters from domain to schema models. `models_library.service_io` are domain models.

![image](https://github.com/user-attachments/assets/e15cdb9b-5c22-4669-99c6-8c123a3a6869)